histomicstk/features/ComputeNucleiFeatures.py

```diff
@@ -167,6 +180,26 @@ def ComputeNucleiFeatures(im_label, im_nuclei, im_cytoplasm=None,
 
         feature_list.append(fgrad_cytoplasm)
 
+    # compute nuclei haralick features
+    if haralick_features_flag:
+
+        fharalick_nuclei = ComputeHaralickFeatures(im_label, im_nuclei,
```

Comment:
@cooperlab [ComputeHaralickFeatures](https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/histomicstk/features/ComputeHaralickFeatures.py) has the following parameters:

    offsets : array_like, optional
        A (num_offsets, num_image_dims) array of offset vectors
        specifying the distance between the pixel-of-interest and
        its neighbor. Note that the first dimension corresponds to
        the rows.
        Because this offset is often expressed as an angle, the
        following table lists the offset values that specify common
        angles for a 2D image, given the pixel distance D.
        AngleXY  |  OFFSET
        -------  |  ------
        0        |  [0 D]
        45       |  [-D D]
        90       |  [-D 0]
        135      |  [-D -D]
        Default
            [1] for 1D,
            [[1, 0], [0, 1]] for 2D,
            [[1, 0, 0], [0, 1, 0], [0, 0, 1] for 3D and so on
    num_levels : unsigned int, optional
        An integer specifying the number of gray levels For example, if
        `num_levels` is 8,  the intensity values of the input image are
        scaled so they are integers between 1 and 8.  The number of gray
        levels determines the size of the gray-level co-occurrence matrix.
        Default: 8 for numeric image, 2 for binary/logical image
    gray_limits : array_like, optional
        A two-element array specifying the desired input intensity range.
        Intensity values in the input image will be clipped into this range.
        Default: [0, 1] for binary/logical image, [0, 255] for numeric image

What do you think would be good parameters for nuclei classification? 

Not sure if gray_limits [0, 255] is good for nuclei. The intensity range of nuclear/cytoplasm pixels might be much more narrow than [0, 255].

num_levels is set by default to log_base_2( max_intensity=256 ). 

what offsets should we use? By default it uses `offsets=[ [0, 1], [1, 0] ]` for 2D. I feel we should at least add the 45 degree offset.

CC: @slee172